pkg/apis/sriovnetwork/v1/sriovnetworknodepolicy_types.go

```diff
@@ -33,6 +33,9 @@ type SriovNetworkNodePolicySpec struct {
 	DeviceType string `json:"deviceType,omitempty"`
 	// RDMA mode. Defaults to false.
 	IsRdma bool `json:"isRdma,omitempty"`
+	// +kubebuilder:validation:Enum=eth;ETH;ib;IB
+	// NIC Link Type. Allowed value "eth", "ETH", "ib", and "IB".
```

Comment:
I think you should swap the comments so that CRD generation adds the enum to the schema

Reply:
It is adding the enum like that

Reply:
missed that :) NVM.